### PR TITLE
Create feature flag for SE-0384

### PR DESCRIPTION
**Pull request by NuriAmari**

### CODE REVIEW

**Comment by NuriAmari** [comment]
@swift-ci Please test

**Comment by NuriAmari** [comment]
@swift-ci Please test

File: lib/AST/ASTPrinter.cpp [hunk 1]
```diff
@@ -3159,6 +3159,24 @@ static bool usesFeatureExistentialAny(Decl *decl) {
   return false;
 }
 
+static bool usesFeatureImportObjcForwardDeclarations(Decl *decl) {
```
Comment (allevato) [inline lib/AST/ASTPrinter.cpp:3162]: Is this being used for anything? I don't see it referenced elsewhere in this PR.

Comment (NuriAmari) [reply lib/AST/ASTPrinter.cpp:3162]: Yes, but not obviously -- above in the same file:

```
#define LANGUAGE_FEATURE(FeatureName, SENumber, Description, Option)  \
    if (usesFeature##FeatureName(decl))                               \
      collectRequiredFeature(Feature::FeatureName, operation);
#define SUPPRESSIBLE_LANGUAGE_FEATURE(FeatureName, SENumber, Description, Option)  \
    if (usesFeature##FeatureName(decl))                               \
      collectSuppressibleFeature(Feature::FeatureName, operation);
#include "swift/Basic/Features.def"
  }
};
```
It won't compile without this.

Comment (allevato) [reply lib/AST/ASTPrinter.cpp:3162]: Ah, metaprogramming. Thanks for clarifying!

**Review by allevato — APPROVED** [review]
Thanks for the quick turnaround on this! Once this is merged, can you please create a cherry-pick into the 5.9 branch, and also update the text of SE-0384 to refer to the feature name (and remove the reference to the old flag)?

**Pull request merged**